src/cmd/usb-util.js

```diff
@@ -124,8 +124,11 @@ async function executeWithUsbDevice({ args, func, dfuMode = false } = {}) {
 	} finally {
 		if (deviceIsProtected) {
 			device = await reopenDevice(device);
-			// FIXME: ignore errors
-			await deviceProtectionHelper.turnOffServiceMode(device);
+			try {
+				await deviceProtectionHelper.turnOffServiceMode(device);
+			} catch (error) {
+				// FIXME: ignore errors
+			}
 		}
```

Comment:
`executeWithUsbDevice` is called from `forEachUsbDevice`, and this error will be propagated to `forEachUsbDevice` and if not handled properly, it will show an internal error to the user (such as `IN control transfer failed` which is likely useless to the user because they cant act on it)